stable/opsportal/templates/landing.yaml

```diff
@@ -51,7 +51,7 @@ spec:
     spec:
       containers:
         - name: opsportal-landing
-          image: nginx:1.15-alpine
+          image: nginx:1.17-alpine
```

Comment:
There's a known issue with [Alpine based container images](https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/#known-issues) but I guess this is fine because it's only used to serve static content.